src/TLuaInterpreter.cpp

```diff
@@ -1628,6 +1628,42 @@ int TLuaInterpreter::selectSection(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#getSelection
+int TLuaInterpreter::getSelection(lua_State* L)
+{
+    bool valid;
+    QString text;
+    int start, length;
+
+    auto& host = getHostFromLua(L);
+
+    QString windowName;
+    if (lua_gettop(L) > 1) { // Have more than one argument so first must be a console name
```

Comment:
Wouldn't this be "at least one argument"?

Reply:
Yep though it's not a useful comment all in all, you can read the code to and get what it means easily.

Reply:
But don't you also have to compare with `> 0` because you don't expect any other arguments? Or is my logic screwed up here?

Reply:
I think you're right. I didn't test the other window case.

Reply:
Fixed.